lib/matplotlib/tests/test_axes.py

```diff
@@ -4343,15 +4376,15 @@ def test_rc_tick():
         xax = ax1.xaxis
         yax = ax1.yaxis
         # tick1On bottom/left
-        assert xax._major_tick_kw['tick1On'] == False
-        assert xax._major_tick_kw['tick2On'] == True
-        assert xax._minor_tick_kw['tick1On'] == False
-        assert xax._minor_tick_kw['tick2On'] == True
+        assert not xax._major_tick_kw['tick1On']
```

Comment:
These are slightly different tests as `assert 'aardvark'` passes, but `assert 'aardvark' == True` does not.  

Do we care about this change?